template/package.json

```diff
@@ -13,8 +13,11 @@
     {{#e2e}}
     "e2e": "node test/e2e/runner.js",
     {{/e2e}}
-    "test": "{{#unit}}npm run unit{{/unit}}{{#unit}}{{#e2e}} && {{/e2e}}{{/unit}}{{#e2e}}npm run e2e{{/e2e}}"{{#lint}},
-    "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"{{/lint}}
+    "lint": "{{#eslint}}npm run lint:eslint{{/eslint}}{{#eslint}}{{#stylelint}} && {{/eslint}}{{/stylelint}}{{#stylelint}}npm run lint:stylelint{{/stylelint}}",
+    "test": "{{#unit}}npm run unit{{/unit}}{{#unit}}{{#e2e}} && {{/e2e}}{{/unit}}{{#e2e}}npm run e2e{{/e2e}}"{{#eslint}},
+    {{#eslint}}
+    "lint:eslint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"{{/eslint}}{{#stylelint}},
+    "lint:stylelint": "stylelint 'src/**/*.@(?(s)?(a|c)ss|vue|html)' --config ./.stylelintrc.js --extract"{{/stylelint}}
```

Comment:
It looks like stylelint can also lint [scss and less](https://github.com/stylelint/stylelint/blob/2db77ded9be9e53e1c56ba52db42a2cad4c24bcd/src/__tests__/integration.js#L86-L118), but it needs to be run again specifying that extension and syntax. See [these usage examples](https://github.com/stylelint/stylelint/blob/2db77ded9be9e53e1c56ba52db42a2cad4c24bcd/docs/user-guide/cli.md#usage-examples).

Another question - what will happen with if `.vue` files use a transpile-to-css language?